docs/accounts/billing.md

```diff
@@ -1,9 +1,12 @@
 # Billing
 
+See the [Billing unit and price calculator](https://research.csc.fi/billing-and-monitoring#buc) at research.csc.fi.
 
-## Puhti 
+## Accounting formulas
```

Comment:
Do we really need another header level here? Everything is under this one, seems superfluous

Reply:
You are right. I'll remove the extra header.
I added it when I tried to embed the billing unit calculator, but forgot to remove it.